docs/projects/reviews/0000-template-annual.md

```diff
@@ -0,0 +1,39 @@
+!!! info Instructions
+    _Copy this template to the subdirectory for the current year and name the file `YYYY-Project-Name-annual.md` (e.g., `2024-amazingproj-annual.md`). Update the `index.md` file in the current year to include a link to the markdown file. These blocks are instructions. Please remove when section has been completed._
+
+# Project Health
+
+!!! info Instructions
+    _Include a link to your project’s [LFX Insights page](https://insights-v2.lfx.linuxfoundation.org/projects). We will be looking for signs of consistent or increasing contribution activity. Please feel free to add commentary to add color to the numbers and graphs we will see on Insights._
+
+# Maintainer Diversity
+
+!!! info Instructions
+    _How many maintainers do you have, and which organisations are they from? How has the maintainers and diversity of your maintainers changed in the past year? Has the number of active maintainers increased/decreased? Has the diversity of maintainers increased/decreased? Please include a link to your existing MAINTAINERS file and the MAINTAINERS file from last year (if appropriate). This is a good opportunity to ensure that your MAINTAINERS file is up to date and to retire any maintainers._ 
```

Comment:
It seems we will be linking to a file in GitHub that points to various commits. It is crucial to emphasize to all our projects the importance of maintaining such a file. 

For instance, the `sd-jwt-python` repository lacks this file.
* https://github.com/openwallet-foundation-labs/sd-jwt-python